src/pytkdocs/parsers/docstrings/google.py

```diff
@@ -9,6 +9,8 @@
     "arguments:": Section.Type.PARAMETERS,
     "params:": Section.Type.PARAMETERS,
     "parameters:": Section.Type.PARAMETERS,
+    "keyword args:": Section.Type.PARAMETERS,
+    "keyword arguments:": Section.Type.PARAMETERS,
```

Comment:
I think that for the sake of rendering with a different title than "Parameters" in mkdocstrings, we will need a new section type. Probably `Section.Type.KEYWORD_PARAMETERS`. It means you'll also have to add an key-value pair in the parser `section_reader` attribute. We can still reuse the same `read_parameters_section` though :slightly_smiling_face:

Reply:
Ah, the method returns hardcodes a `Section.Type.PARAMETERS` though. Maybe accept a new arg in the method, and set partials in `section_reader`. Or refactor the method into three (one that does almost everything while the other two just create the actual section with the right type).

Reply:
I was thinking about this. I checked how the other parsers were done and i.e. for rst the args are the same: `PARAM_NAMES = frozenset(("param", "parameter", "arg", "argument", "key", "keyword"))`, so I thought that for g-style the same would apply.

However, having a different proper Section type would certainly be better. And I think that unhardcoding is the way to go so I'm going to refactor the `read_parameters_section` method into three.

Reply:
Changes pushed